modin/logging/config.py

```diff
@@ -117,8 +117,19 @@ def _create_logger(
     Logger
         Logger object configured per Modin settings.
     """
-    log_filename = f".modin/logs/job_{job_id}/{log_name}.log"
-    os.makedirs(os.path.dirname(log_filename), exist_ok=True)
+
```

Comment:
```suggestion
```

Reply:
With that being said, it's NBD and I'm fine either way.

Reply:
I see. It's one of the checks. My bad.